docs/ta/docs/index.md

```diff
@@ -0,0 +1,463 @@
+
+### ஊடாடும் API ஆவணம்
+
+இப்பொழுது <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a> தளத்திற்க்கு செல்லுங்கள்.
+
+அங்கு தானியங்கி ஊடாடும் API ஆவணம் (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>யால் வழங்கப்படுகிறது) பாரப்பிர்கள்:
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+
+### மாற்று API ஆவணம்
+
+<a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a> தளத்திற்க்கு சென்றால், (<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>க்கால் வழங்கப்படுகிறது) மாற்று தானியங்கி ஆவணத்தை காண்பீர்:
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
+
+## உதாரண மேம்படுத்தல்
+
+இப்பொழுது, `main.py` கோப்பையை ஒரு `PUT` வேண்டுதலில் இருந்து உடலை பெறுவதற்க்கு மாற்றியமைக்கலாம்.
+
+உடலை செந்தரம் பைத்தான் வகைகளை கொண்டு அறிவிக்கலாம், Pydanticக்கு நன்றி.
+
+```Python hl_lines="4  9-12  25-27"
+from typing import Union
+
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+app = FastAPI()
+
+
+class Item(BaseModel):
+    name: str
+    price: float
+    is_offer: Union[bool, None] = None
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/items/{item_id}")
+def read_item(item_id: int, q: Union[str, None] = None):
+    return {"item_id": item_id, "q": q}
+
+
+@app.put("/items/{item_id}")
+def update_item(item_id: int, item: Item):
+    return {"item_name": item.name, "item_id": item_id}
+```
+
+சேவையகம் தானாகவே புதுப்பிக்கும் (ஏனென்றால், முன்பு `uvicorn` கட்டளையில் `--reload` சேர்த்திறிக்கிறோம்).
+
+### ஊடாடும் API ஆவணம் மேம்படுத்தல்
+
+இப்பொழுது <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a> தளத்திற்க்கு செல்லுங்கள்.
+
+* ஊடாடும் API ஆவணம், தானாகவே புதுப்பதிற்க்கும், புது உடலுடன்::
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-03-swagger-02.png)
+
+* "Try it out" பட்டனை சொடக்கினால், அளவுருவிக்களை நிரப்பி, APIயுடன் தொடர்புக்கொள்ள அனுமதிக்கும்:
+
+![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-04-swagger-03.png)
+
+* "Execute" பட்டனை சொடக்கினால், பயனாளர் இடைமுகப்பு APIயுடன் தொடர்புக்கொண்டு, அளவுருவிகளை அனுப்பி, முடிவுகளை பெற்று திரையில் காண்பிக்கும்.
+
+![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-05-swagger-04.png)
+
+### மாற்று API ஆவணம் மேம்படுத்தல்
+
+ இப்பொழுது <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>  தளத்திற்க்கு செல்லுங்கள்.
+
+* மாற்று ஆவணமும் புது உடல் மற்றும் கேட்டறி அளவுருவி காண்பிக்கும்:
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-06-redoc-02.png)
+
+### மீண்டும்
+
+சுருக்கமாக, அளவுருவியின் வகை, உடல் போன்றவற்றை **ஒரு முறை** செயற்ப்பணி அளவுருவியாக அறிவித்தல்.
+
+அதை செந்தரம் நவீன பைத்தான் வகைகள் மூலமாக செய்ய வேண்டும்.
```

Comment:
For `standard`, `செந்தரம்` is used but we need a more precise word. Some choices: நிலை, விதிமுறை

Reply:
@arunppsg I would suggest not to use prose version of Tamil, as it would need more help for understanding. If you can use the form of Tamil which is in regular use(with simple words), I will be great. I am interested in helping you also.

Reply:
Updated the usage to simple terms. Hope it is better now!